### ISSUE TITLE
Clicking Meridiem buttons can sometimes change selected minute.

### ISSUE THREAD

**Issue by mac-s-g** [catc/react-timekeeper#73]
first and foremost - **thank you** to the authors for building an amazing component and sharing it with the community.

I encountered an intermittent issue where attempting to click the circular meridiem buttons unintentionally changes the selected hour or minute value.

here's an example:

https://user-images.githubusercontent.com/4097374/126244200-faa47d83-a07e-49b7-9800-b98f1ccc93f9.mp4

I eventually realized the problem occurs when my mouse drags during the button click.

In other words, 1 or more pixels of mouse movement between mouse down and mouse up causes the intended meridiem button click to be interpreted as a clock hand drag.

The issue can be hard to avoid when mouse sensitivity is high.

One possible solution would be to disable clock hand drag when the target of mouse down is a meridiem button.
